### ISSUE TITLE
Launch process of Azure Cloud Shell when expiring access token.

### ISSUE THREAD

**Issue by horihiro** [microsoft/terminal#9766]
# Description of the new feature/enhancement
When the access token for Azure Cloud Shell expires, showing log-in prompt is wanted.

Currently, only an error message which means the token is already expired is shown as the following screenshot if the token is expired.
So the user needs to remove the connection setting once.
![image](https://user-images.githubusercontent.com/4566555/114258890-e22bcc80-9a04-11eb-85c8-e770d126aeae.png)

# Proposed technical implementation details (optional)

If the token is expired, it's better that the login process restarts again instead of showing the current error message.
Or it's more appropriate that the messages indicate that the connection settings have to be deleted once.

Could you please consider that?

Thanks.

**Comment by Don-Vito** [comment]
What you suggest can be an interesting workaround, but the real fix should refresh the token on your behalf if the session is active.

**Comment by horihiro** [comment]
Thank you Don-VIto,

Do you have a plan to fix this as you mentioned?

**Comment by DHowett** [comment]
Yeah, we definitely thought that we were handling token expiration correctly. Thanks for reporting this!

**Comment by Don-Vito** [comment]
> Thank you Don-VIto,
> 
> Do you have a plan to fix this as you mentioned?

Probably I am wrong, but we might need to introduce a "control channel" to fix this. There should be some ticket talking about this already. 

@DHowett - do you have a contact in posh (anywhere else) who can advice, so we don't reinvent the wheel. Alternatively I can try to reach out to Identity (msal?) folks.
